src/components/App.js

```diff
@@ -1,6 +1,7 @@
 import { css, html, LitElement } from "lit";
 import { customElement } from "lit/decorators.js";
 import { ref } from "lit/directives/ref.js";
+import { fetchTopics } from "../api";
```

Comment:
Usar las ruta absolutas queda un poco más limpio.
```suggestion
import { fetchTopics } from "/api";
```

Reply:
Copiado :eye: